src/main/java/io/orten/nano/impl/SwishImplementation.java

```diff
@@ -0,0 +1,165 @@
+package io.orten.nano.impl;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.orten.nano.model.SwishPaymentRequestObject;
+
+import javax.net.ssl.*;
+import java.io.*;
+import java.net.URL;
+import java.net.URLConnection;
+import java.security.KeyStore;
+import java.security.SecureRandom;
+import java.security.cert.X509Certificate;
+import java.util.Map;
+
+
+public class SwishImplementation {
+
+    public static URLConnection createURL(String url) throws Exception{
+        URL myURL = new URL(url);
+        URLConnection connection = myURL.openConnection();
+        connection.setDoOutput(true);
+        return connection;
+    }
+
+    public static void writeObject(InputStream input) throws IOException {
```

Comment:
confusing name as it can be interpreted as write object to swish and not System.out.

Also, is this function really needed?
If all it does is print to System.out, then it can probably be removed